lib/reporters/base.js

```diff
@@ -219,6 +219,11 @@ exports.list = function(failures) {
     stack = stack.replace(/^/gm, '  ');
 
     console.log(fmt, (i + 1), test.fullTitle(), msg, stack);
+
+    // [hack] Handle waterline's nested `originalError`
```

Comment:
_(I tried sticking this onto `stack` higher up, but that didn't format correctly.)_